packages/svelte-urql/package.json

```diff
@@ -43,7 +43,7 @@
     "check": "tsc --noEmit",
     "lint": "eslint --ext=js,jsx,ts,tsx .",
     "build": "rollup -c ../../scripts/rollup/config.js",
-    "prepare": "../../scripts/prepare/index.js",
+    "prepare": "node ../../scripts/prepare/index.js",
```

Comment:
Thank for fixing this. Must’ve been an oversight as we only test on Windows occasionally.
Can you update this in all package.json files please, if it’s not too much trouble? 🙌

Reply:
Sure. One minute.